packages/block-library/src/post-featured-image/edit.js

```diff
@@ -44,7 +44,13 @@ function PostFeaturedImageDisplay( {
 	);
 	const media = useSelect(
 		( select ) =>
-			featuredImage && select( coreStore ).getMedia( featuredImage ),
+			featuredImage &&
+			select( coreStore ).getEntityRecord(
```

Comment:
@youknowriad, is it possible to pass additional params to `getMedia` and other methods automatically generated from entities? One thing that surprises me is that I see `getEntityRecord` used so often despite all thos shorthands available:

<img width="383" alt="Screen Shot 2021-08-06 at 08 16 11" src="https://user-images.githubusercontent.com/699132/128465031-ba01408f-2d34-4176-8dea-be6e8ee6cd78.png">

Reply:
@gziolo, It wasn't possible when I created this PR, but I did follow up one as well for shorthands #33689.

I can create PR to switch bach to `getMedia`.

Reply:
@Mamaduka, it's optional. I was rather sharing a general thought about the surprising popularity of more verbose `getEntityRecord`.

Reply:
@gziolo, I also kinda like `getMedia( mediaId, { context: 'view' } )` more 😄  So follow-up PR won't be a problem.